mne/decoding/tests/test_search_light.py

```diff
@@ -55,6 +57,13 @@ def test_searchlight():
     assert_true(np.sum(np.abs(score)) != 0)
     assert_true(score.dtype == float)
 
+    # change score method
+    sl = SearchLight(LogisticRegression(), scoring=roc_auc_score)
+    sl.fit(X, y)
+    score = sl.score(X, y)
```

Comment:
add explicit test that check the score 

`assert_equal(score[0], roc_auc_score(y, LogisticRegression().fit(X,y).predict_proba(X)[:, 1))`